geps/gep-3792/index.md

```diff
@@ -0,0 +1,190 @@
+# GEP-3792: External Gateways
+
+* Issue: [#3792](https://github.com/kubernetes-sigs/gateway-api/issues/3792)
+* Status: Provisional
+
+(See [status definitions](../overview.md#gep-states).)
+
+## User Story
+
+**[Chihiro] and [Ian] want a way for out-of-cluster Gateways to be able to
+usefully participate in a GAMMA-compliant in-cluster service mesh.**
+
+Historically, API gateways and ingress controllers have often been implemented
+using a Service of type LoadBalancer fronting a Kubernetes pod running a
+proxy. This is simple to reason about, easy to manage for sidecar meshes, and
+will presumably be an important implementation mechanism for the foreseeable
+future. Some cloud providers, though, are moving the proxy outside of the
+cluster, for various reasons which are out of the scope of this GEP. Chihiro
+and Ian want to be able to use these out-of-cluster proxies effectively and
+safely, though they recognize that this may require additional configuration.
+
+[Chihiro]: https://https//gateway-api.sigs.k8s.io/concepts/roles-and-personas/#chihiro
+[Ian]: https://https//gateway-api.sigs.k8s.io/concepts/roles-and-personas/#ian
+
+### Nomenclature and Background
+
+In this GEP, we will use _out-of-cluster Gateway_ (OCG) to refer to a
+conformant implementation of Gateway API's `GATEWAY` profile that's running
+outside of the cluster. This would most commonly be a managed implementation
+from a cloud provider, but of course there are many other possibilities.
+
+We'll also distinguish between _mTLS meshes_, which rely on standard mTLS for
+secure communications including workload authentication, and _non-mTLS
+meshes_, which do anything else. We'll focus on mTLS meshes in this GEP; this
+isn't because of a desire to exclude non-mTLS meshes, but because we'll have
+enough trouble just wrangling the mTLS meshes! Supporting non-mTLS meshes will
+be a separate GEP.
+
+Finally, _authentication_ is the act of verifying the identity of some
+_principal_; what the principal actually is depends on context. For this GEP
+we will primarily be concerned with _workload authentication_ (also known as
+_east/west auth_), in which the principal is a workload. (For completeness'
```

Comment:
How can there be east/west auth **outside** of the cluster?  Does the OCG represent a connection point between meshes on multiple clusters?  Can we add such a "connection point" to Goals/Non-Goals to clarify?

Reply:
The OCG must properly authenticate itself, as a workload, to the mesh to be able to usefully participate in the mesh. This is workload auth. Workload auth is sometimes called east/west auth, in the same way that user auth is sometimes called north/south auth,  wherever it's actually happening.

I can definitely add that multicluster is currently a non-goal.

Reply:
Yanked directional terms, and added multicluster as a non-goal.